pkg/operator/ceph/cluster/controller.go

```diff
@@ -400,7 +385,7 @@ func (c *ClusterController) onUpdate(oldObj, newObj interface{}) {
 		return
 	}
 
-	logger.Debugf("update event for cluster %s", newClust.Namespace)
+	logger.Infof("update event for cluster %s", newClust.Namespace)
```

Comment:
I'd like to leave these "update event" logs as debug. Since the ceph status is updated every minute now, it fills the operator log.